app_dart/bin/server.dart

```diff
@@ -36,7 +37,9 @@ Future<void> main() async {
       '/api/push-build-status-to-github': PushBuildStatusToGithub(config, authProvider),
       '/api/push-engine-build-status-to-github': PushEngineStatusToGithub(config, authProvider),
       '/api/refresh-chromebot-status': RefreshChromebotStatus(config, authProvider),
-      '/api/refresh-github-commits': RefreshGithubCommits(config, authProvider),
+      '/api/refresh-github-commits': RefreshGithubCommits(config, authProvider,
+          tabledataResourceApi:
+              await BigqueryService(accessClientProvider).defaultTabledata()),
```

Comment:
nit: This should format more nicely if you add a `,` after `defaultTabledata()`.

Reply:
Have re-written the api.